### ISSUE TITLE
GBFS spider for bike and car sharing

### ISSUE THREAD

**Issue by brawer** [alltheplaces/alltheplaces#4216]
Would bike and car sharing stations be within scope of AllThePlaces?

If so, one could write a spider for the [GBFS feed format](https://github.com/MobilityData/gbfs/blob/v2.0/gbfs.md) that fetches all feeds listed in [systems.csv](https://github.com/MobilityData/gbfs/blob/v2.0/systems.csv). It would be nice to make this extensible. For example, the Swiss public administration [offers an open data feed](https://github.com/SFOE/sharedmobility) that aggregates [all shared mobility stations in Switzerland](https://sharedmobility.ch/) (including car sharing, taxi stands, etc.) in GBFS format, using a [custom GBFS extension](https://github.com/SFOE/sharedmobility/blob/main/Additions%20to%20GBFS.md) to express vehicle type and brand/operator. Similar GBFS extensions may exist for other countries but I haven’t checked.

I think this would be quite useful for maintaining OpenStreetMap. For that purpose, it would be sufficient to emit just the non-realtime attributes, ie. the tag combinations recommended for [`amenity=bicycle_rental`](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbicycle_rental) respectively [`amenity=car_sharing`](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcar_sharing).

**Comment by Cj-Malone** [comment]
I've worked with this data before, and imported some of the GB ones to OSM.

I've got the barebones of a spider to do exactly this, get the points from all the entries in systems.csv.

I'd be happy to do a PR, @iandees will have to decide if it's in scope, it's public transit like the NAPTaN one I recently wrote. About 75k pois.

**Comment by iandees** [comment]
I think I'm ok with this data in here as long as it's coming from the first party and not an aggregator.

**Comment by Cj-Malone** [comment]
Great, so [this](https://github.com/MobilityData/gbfs) is an aggregator of APIs, but those APIs are first party data. TBH I think this is the perfect way for us, we don't have to maintain a list of providers but we can do PRs if needed. I already did one to update my local provider.

@brawer did you have any code/want to do a PR? Like I said, I've already got some code but it's the bare minimum, it could do with more work so follow up PRs would be appreciated.
